src/core/lib/transport/metadata_batch.h

```diff
@@ -22,19 +22,38 @@
 #include <grpc/support/port_platform.h>
 
 #include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
 
+#include <algorithm>
 #include <cstdint>
 #include <limits>
+#include <string>
+#include <type_traits>
+#include <utility>
+
+#include "absl/container/inlined_vector.h"
```

Comment:
Didn't you just remove a bunch of these includes from this file?

Reply:
Yeah, absolutely.  Just wanted to make sure this wasn't accidenally undoing something you already did.